src/compiler/expression_error.rs

```diff
@@ -40,14 +41,14 @@ impl From<ExpressionError> for Diagnostic {
     }
 }
 
-impl DiagnosticMessage for ExpressionError {
+impl DiagnosticMessage for ExpressionError2 {
```

Comment:
This seems like a temporary name, is it supposed to change to something else, or are we keeping it for a bit?

Reply:
Addressed here: https://github.com/vectordotdev/vrl/pull/554